src/main/java/eu/mihosoft/vrl/v3d/Text3d.java

```diff
@@ -91,22 +73,22 @@ public Text3d(String text, double depth) {
      * @param depth text depth (z thickness)
      */
     public Text3d(String text, String fontName, double fontSize, double depth) {
-        this.text = text;
-        this.fontName = fontName;
-        this.size = fontSize;
-        this.depth = depth;
 
-        // scaled size and height (see scaleFactor docs)
-        int realSize = (int) (fontSize * scaleFactor);
-        double realHeight = depth * scaleFactor;
-
-        t3dMesh = new Text3DMesh(
-                text, fontName, realSize, false, realHeight, 0, 0);
+    	Font font = new Font(fontName,  Font.PLAIN, (int) fontSize);
+    	letters = TextExtrude.text( depth,  text,  font);
+    	for (int i=0;i<letters.size();i++){
+    		letters.set(i, letters.get(i)
```

Comment:
Awt test spits out a path iterator, and crawling that iterator is how the points for the extrude are extracted onto a list of points to be passed to the extrude function. I will look into the JavaFX fonts and see if a PathIterator can be pulled out of it. If not I'll separate out the helper functions into a stand alone pr so I can get the Bezier and SVG loading mainlined, even if text has to take a bit more work to get up to speed.